source/sdk/lang/Exception.ooc

```diff
@@ -34,6 +34,8 @@ exceptionStack := ThreadLocal<Stack<StackFrame>> new()
 _exception := ThreadLocal<Exception> new()
 _EXCEPTION: Int = 1
 
+stackFramesToDelete := Stack<StackFrame> new()
```

Comment:
Let's clear this up in another PR / debt issue, along with all other global stuff in Exception / Backtrace. What do you think ?